src/blocks/control.js

```diff
@@ -383,22 +383,9 @@ Blockly.Blocks["control_start_as_clone"] = {
 
 Blockly.Blocks["control_create_clone_of_menu"] = {
   /**
-   * Create-clone drop-down menu.
+   * Create-clone drop-down menu. Populated dynamically by scratch-gui.
```

Comment:
If this is going to be created by scratch-gui, and is unusable without scratch-gui code running, it seems better to not add it to `Blockly.Blocks` at all, and make scratch-gui responsible for that as well.

Reply:
Even if you want to keep these, you don't need the `@this` anymore, and can therefore move the comment to above `Blockly.Blocks["name"] = {};` and compress the declaration down to a single line.

Reply:
Condensed them accordingly; I think ultimately I'd like to pass a reference to the VM into scratch-blocks so that all the definitions can be here.